bndtools.core/src/bndtools/wizards/project/ImportBndWorkspaceWizard.java

```diff
@@ -0,0 +1,438 @@
+
+    private void importConfigurationProject(File rootImportPath, IProgressMonitor monitor) throws Exception {
+        monitor.subTask("Import configuration project 'cnf'.");
+        final IWorkspace eclipseWorkspace = ResourcesPlugin.getWorkspace();
+        final IWorkspaceRoot eclipseWorkspaceRoot = eclipseWorkspace.getRoot();
+        final Workspace bndWorkspace = Workspace.getWorkspace(rootImportPath);
+
+        // Prepare Eclipse Workspace
+        updateEclipseWorkspaceSettings(bndWorkspace);
+
+        // create generic project
+        final IProjectDescription cnfProjectDescription = eclipseWorkspace.newProjectDescription(Workspace.CNFDIR);
+        final IProject project = eclipseWorkspaceRoot.getProject(Workspace.CNFDIR);
+
+        deleteOldProjectFiles(Paths.get(bndWorkspace.getBase().toURI()).resolve(Workspace.CNFDIR));
+
+        // create JavaProject
+        IPath path = URIUtil.toPath(bndWorkspace.getBuildDir().toURI());
+        if (Platform.getLocation().isPrefixOf(path)) {
+            cnfProjectDescription.setLocation(null);
+        } else {
+            cnfProjectDescription.setLocation(path);
+        }
+
+        project.create(cnfProjectDescription, monitor);
+        project.open(monitor);
+
+        project.setDefaultCharset("UTF-8", monitor);
+
+        monitor.worked(1);
+    }
+
+    private void importBndProject(Project bndProject, Workspace bndWorkspace, IProgressMonitor monitor) throws IOException, CoreException, Exception {
+        monitor.subTask("Import Bnd project '" + bndProject.getName() + "'.");
+        final IWorkspace eclipseWorkspace = ResourcesPlugin.getWorkspace();
+        final IWorkspaceRoot eclipseWorkspaceRoot = eclipseWorkspace.getRoot();
+
+        deleteOldProjectFiles(Paths.get(bndWorkspace.getBaseURI()).resolve(bndProject.getName()));
+        // create generic project
+        final IProjectDescription projectDescription = eclipseWorkspace.newProjectDescription(bndProject.getName());
+        final IProject project = eclipseWorkspaceRoot.getProject(bndProject.getName());
+
+        IPath path = URIUtil.toPath(bndProject.getBaseURI());
+        if (Platform.getLocation().isPrefixOf(path)) {
+            projectDescription.setLocation(null);
+        } else {
+            projectDescription.setLocation(path);
+        }
+
+        project.create(projectDescription, monitor);
+        project.open(monitor);
+        project.setDefaultCharset("UTF-8", monitor);
+
+        setNatures(project, monitor, JavaCore.NATURE_ID, Plugin.BNDTOOLS_NATURE);
+
+        // create JavaProject
+        IJavaProject javaProject = JavaCore.create(project);
+
+        javaProject.open(monitor);
+        updateJavaProjectSettings(bndProject, javaProject, monitor);
+
+        importSourceAndOutputFolders(bndProject, project, javaProject, monitor);
+
+        // create BndToolsProject
+        // TODO why create BndProject?
+        BndProject bndtoolsProject = new BndProject(javaProject.getProject());
```

Comment:
I dont know :-) In my tests I did not but I wasnt sure if this might trigger something else.
Will remove it